src/main/java/org/terasology/launcher/gui/javafx/ApplicationController.java

```diff
@@ -362,6 +371,7 @@ private void initComboBoxes() {
             selectedPackage = newVal.getLinkedPackage();
 
             if (selectedPackage != null && selectedPackage.isInstalled()) {
+                selectedPackageProperty.setValue(selectedPackage);
```

Comment:
there's not an existing property we can use quite so easily for `selectedPackageProperty`, as it had to go through the selected version's getLinkedPackage ... okay, maybe that means adding a new Property for it is appropriate.

what if there is no selected version, or no package for the selected version, or there is a package but it's not installed? what should selectedPackageProperty be in those cases?

Some of those are probably very unlikely, but I think others might come up often when you do things like change between the different build types, e.g. the "(nightly)" builds.